jq.py

```diff
@@ -149,12 +152,17 @@ def outputDataAsString(debugger, symbol):
     interpreter = debugger.GetCommandInterpreter()
```

Comment:
Add another parameter to this method: is_optional

If this parameter is true, check for nil, otherwise, don't bother to check for nil; that can never happen

If it's true, add the '!' to force unwrap the Data in `commandString`, otherwise, don't force unwrap